themes/bootstrap5/js/channels.js

```diff
@@ -1,258 +1,329 @@
-/*global bootstrap, getUrlRoot, VuFind */
-VuFind.register('channels', function Channels() {
+/*global getUrlRoot, VuFind */
+VuFind.register("channels", function Channels() {
   /**
-   * Add dropdown link buttons to an element based on a JSON data attribute.
-   * @param {HTMLElement} elem The element containing the data-link-json attribute.
+   * @param {HTMLElement} channel Channel to search
```

Comment:
Some of the functions in this file lack top-level descriptive comments. For example, something like this:

```suggestion
   * Retrieve a specific item's element from within a channel.
   * @param {HTMLElement} channel Channel to search
```